homeassistant/components/nut/const.py

```diff
@@ -0,0 +1,135 @@
+"""The nut component."""
+from homeassistant.const import POWER_WATT, TEMP_CELSIUS, TIME_SECONDS, UNIT_PERCENTAGE
+
+DOMAIN = "nut"
+
+PLATFORMS = ["sensor"]
+
+
+DEFAULT_NAME = "NUT UPS"
+DEFAULT_HOST = "localhost"
+DEFAULT_PORT = 3493
+
+KEY_STATUS = "ups.status"
+KEY_STATUS_DISPLAY = "ups.status.display"
+
+PYNUT_DATA = "data"
+PYNUT_STATUS = "status"
+PYNUT_UNIQUE_ID = "unique_id"
+PYNUT_MANUFACTURER = "manufacturer"
+PYNUT_MODEL = "model"
+PYNUT_FIRMWARE = "firmware"
+
+SENSOR_TYPES = {
+    "ups.status.display": ["Status", "", "mdi:information-outline"],
+    "ups.status": ["Status Data", "", "mdi:information-outline"],
+    "ups.alarm": ["Alarms", "", "mdi:alarm"],
+    "ups.time": ["Internal Time", "", "mdi:calendar-clock"],
+    "ups.date": ["Internal Date", "", "mdi:calendar"],
```

Comment:
We shouldn't store relative times in the state machine, as it means that we get a state update on each update. Instead, we should store the timestamp + give it right device class, and in frontend you can choose how to show it. 

For current time/date, I think that we can drop them?

Reply:
This is existing so it would make it a breaking change, but can drop them for sure.

Reply:
nuked as well